include.js

```diff
@@ -1577,18 +1577,11 @@ let cards = [
     githubLink: 'https://github.com/psykat1116'
   },
   {
-    artName: 'Square Pop up',
-    pageLink: './Art/leonyangela/index.html',
-    imageLink: './Art/leonyangela/index.gif',
-    author: 'leonyangela',
-    githubLink: 'https://github.com/leonyangela'
-  },
-  {
-    artName: 'Tulip Growth',
-    pageLink: './Art/Malvericus/index.html',
-    imageLink: './Art/Malvericus/Tulip_Growth.gif',
-    author: 'Malvika Bhadoriya',
-    githubLink: 'https://github.com/Malvericus'
+    artName: 'Loader Animation',
+    pageLink: './Art/gaurav816/index.html',
+    imageLink: './Art/gaurav816/loader.gif',
+    author: 'Gaurav Saxena',
+    githubLink: 'https://github.com/Gauravsaxena816'
   }
```

Comment:
It looks to me as if you are deleting the entries of other contributors.

Please re-check your changes in `include.js`, and make sure that you do not delete other entries.

I suggest that you [sync your fork to upstream](https://short.pm/forksync).